src/main/scala/info/bethard/timenorm/formal/Types.scala

```diff
@@ -494,10 +494,16 @@ case class AfterRI(interval: Interval,
   * @param startInterval first interval
   * @param endInterval   second interval
   */
-case class Between(startInterval: Interval, endInterval: Interval) extends Interval {
+case class Between(startInterval: Interval, endInterval: Interval, startIncluded: Option[String], endIncluded: Option[String]) extends Interval {
```

Comment:
These parameters should be of type Boolean. Do the conversion in Readers; see above.